{"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport/issues/89/comments/45378954.json"}, "html": {"href": "#!/ignitionrobotics/ign-transport/issues/89#comment-45378954"}}, "issue": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport/issues/89.json"}}, "type": "issue", "id": 89, "repository": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport.json"}, "html": {"href": "#!/ignitionrobotics/ign-transport"}, "avatar": {"href": "data/bytebucket.org/ravatar/{4249390a-7f55-404b-990d-817ba94cc7ac}ts=1533306"}}, "type": "repository", "name": "ign-transport", "full_name": "ignitionrobotics/ign-transport", "uuid": "{4249390a-7f55-404b-990d-817ba94cc7ac}"}, "title": "Discovery.hh:RegisterNetIface fails to call IP_ADD_MEMBERSHIP on subinterface"}, "content": {"raw": "Tested by grafting to ign-transport3 which I'm using. I no longer see the error message.\n\nThanks ;)", "markup": "markdown", "html": "<p>Tested by grafting to ign-transport3 which I'm using. I no longer see the error message.</p>\n<p>Thanks ;)</p>", "type": "rendered"}, "created_on": "2018-05-11T13:47:10.469707+00:00", "user": {"display_name": "Martin Pecka", "uuid": "{41691c43-4ae0-49ba-ba45-3c36439f5af2}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B41691c43-4ae0-49ba-ba45-3c36439f5af2%7D"}, "html": {"href": "https://bitbucket.org/%7B41691c43-4ae0-49ba-ba45-3c36439f5af2%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d1ad32a09564f856909994613fdbc37cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsMP-1.png"}}, "nickname": "peci1", "type": "user", "account_id": "557058:434ecab2-e510-4ede-8f9a-1c2a5a4032e1"}, "updated_on": null, "type": "issue_comment", "id": 45378954}